tsconfig.json

```diff
@@ -5,13 +5,12 @@
     "moduleResolution":"node",
     "sourceMap": false,
     "newLine": "LF",
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
     "experimentalDecorators": true,
     "noImplicitUseStrict": true,
     "baseUrl": ".",
-    "lib": ["es5", "es2015.promise", "dom"],
-    "paths": {
-      "N/*": ["N/*"]
```

Comment:
this casues problems with imports, not sure why this is there, but it doesn't make sense to me.